src/psij/web/css/style.css

```diff
@@ -463,17 +492,36 @@ td.run-id {
 
 #status-calendar table {
     position: relative;
+    width:100%;
+    border-spacing: 0;
 }
 
 #status-calendar table tr:first-child th {
     position: sticky;
     top: 0pt;
 }
 
+#status-calendar tr:first-child th {
+    background-color: #666666;
+    color: white;
+    font-weight: bold;
+}
+
+#status-calendar tr td, tr th {
+    background-color: #e2e2e2;
+    border-bottom: 1px solid #cfcfcf;
+}
+
+#status-calendar tr:nth-child(even) td,
+tr:nth-child(even) th {
+    background-color: #f2f2f2;
+}
```

Comment:
May I recommend Butterick's Practical Typography (https://practicaltypography.com/), in particular the sections on tables and grids? It's a good resource in general for creating readable textual content, but it applies in good measure to other things, like websites.